src/smartHandler.test.ts

```diff
@@ -1059,10 +1075,10 @@ describe('authorizeAndFilterReadResponse', () => {
                     fhirUserObject: practitionerFhirResource,
```

Comment:
These test were passing before, because it relies on assumption that practitioner fhir users have system access